agent/lib/kontena/workers/container_info_worker.rb

```diff
@@ -39,6 +39,7 @@ def on_container_event(topic, event)
       container = Docker::Container.get(event.id)
       if container
         self.publish_info(container)
+        self.notify_coroner(container) if container.suspiously_dead?
```

Comment:
I think we want to investigate all.. otherwise master will have ghosts in the db.